rooms/wc.cpp

```diff
@@ -1,8 +1,8 @@
 #include <iostream>
-
+#include <string>
 #include "../main.hpp"
 
-Room wc(__attribute__((unused)) GameData& gameData) {
+Room wc(__attribute__((unused)) GameData &gameData) {
```

Comment:
This should actually be `GameData& gamedata` - the change was unnecessary.

Reply:
```suggestion
Room wc(__attribute__((unused)) GameData& gameData) {
```

Reply:
@dariamagiera ping